lib/Constants.js

```diff
@@ -63,10 +63,10 @@ module.exports.Permissions = {
     manageRoles:          1 << 28,
     manageWebhooks:       1 << 29,
     manageEmojis:         1 << 30,
-    all:      0b1111111111101111111110111111111,
-    allGuild: 0b1111100000000000000000010111111,
+    all:      0b1111111111101111111111111111111,
+    allGuild: 0b1111100000000000000001010111111,
```

Comment:
I don't think the stream permission should be added in `allGuild`. It's a voice-channel only permission, it doesn't have any effect guild-wide.